src/components/RightSidebar/RightSidebar.vue

```diff
@@ -66,13 +67,21 @@
 			icon="icon-settings">
 			<SetGuestUsername />
 		</AppSidebarTab>
+		<AppSidebarTab
```

Comment:
The policy says to avoid using more than 3 tabs.
I think with some other features coming up we discussed to rename projects to advanced, and this could go there then from my pov.